packages/reason-css-parser/lib/Parser.re

```diff
@@ -1707,7 +1707,7 @@ and property_white_space = [%value.rec
 ]
 and property_widows = [%value.rec "<integer>"]
 and property_width = [%value.rec
-  "'auto' | <extended-length> | <extended-percentage> | 'min-content' | 'max-content' | 'fit-content' | fit-content( <extended-length> | <extended-percentage> ) | <calc()>"
+  "'auto' | <extended-length> | <extended-percentage> | 'min-content' | 'max-content' | 'fit-content' | fit-content( <extended-length> | <extended-percentage> ) | <calc()> | <min()> | <max()>"
```

Comment:
No need to add it for property_width since it's already on extended-percentage and extended-length

Reply:
Indeed, thanks